includes/admin/helper/views/html-main.php

```diff
@@ -0,0 +1,188 @@
+<?php defined( 'ABSPATH' ) or exit(); ?>
+
+<div class="wrap wc-helper">
+	<h1><?php _e( 'WooCommerce Helper', 'woocommerce' ); ?></h1>
+
+	<?php include( WC_Helper::get_view_filename( 'html-section-notices.php' ) ); ?>
+	<?php include( WC_Helper::get_view_filename( 'html-section-account.php' ) ); ?>
+
+	<h2>Subscriptions</h2>
```

Comment:
This file is missing localisation on all many of the strings - flagging this one but there are more below.